README.md

```diff
@@ -84,11 +86,12 @@ export AWS_REGION="eu-central-1"
 ```
 
 #### Export boto profile `production`
+
 ```bash
 user> $(aws-export-profile production)
 
 # Validate
-user> env | grep AWS_
+user> printenv | grep AWS_
```

Comment:
I used `env` here instead of `printenv`, because some systems do not have the `printenv` command available.